platform/broadcom/one-image.mk

```diff
@@ -28,7 +28,7 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(ACCTON_AS5812_54X_PLATFORM_MODULE) \
                                $(ACCTON_AS5835_54X_PLATFORM_MODULE) \
                                $(ACCTON_AS9716_32D_PLATFORM_MODULE) \
-                               $(ACCTON_AS5835_54T_PLATFORM_MODULE) \
+			       $(ACCTON_AS5835_54T_PLATFORM_MODULE) \
```

Comment:
you are still modifying this line, do not modify it.

Reply:
Hi sir :
   I fixed the conflict file